.github/workflows/ci.yml

```
@@ -16,17 +20,51 @@ jobs:
+        if: always()
+        with:
+          files: "test-results.xml"
+
+      - name: 'Upload Artifact'
+        if: success()
+        uses: actions/upload-artifact@v4
+        with:
+          name: CellViz
+          path: ./build/CellViz
+          retention-days: 5
+          continue-on-error: true
```

actionlint (1.7.4)

input "continue-on-error" is not defined in action "actions/upload-artifact@v4". available inputs are "compression-level", "if-no-files-found", "include-hidden-files", "name", "overwrite", "path", "retention-days"

(action)